redfish-core/lib/ethernet.hpp

```diff
@@ -2115,27 +2079,6 @@ inline void requestEthernetInterfacesRoutes(App& app)
             }
             });
         });
-
-    BMCWEB_ROUTE(app, "/redfish/v1/Managers/bmc/EthernetInterfaces/<str>/")
```

Comment:
This seems beyond VLAN.. We don't allow deleting ethernet interfaces?

Reply:
@isha309 can you please check why this route is getting deleted?

Reply:
on 1060 as well on patch and get on eth0/eth1 is supported 
https://github.com/ibm-openbmc/bmcweb/blob/1060/redfish-core/lib/ethernet.hpp